packages/devextreme/js/__internal/ui/m_multi_view.ts

```diff
@@ -260,6 +270,10 @@ const MultiView = CollectionWidget.inherit({
     }
   },
 
+  _isItemVisible(index) {
+    return this.option('items')[index].visible ?? true;
```

Comment:
You now have a red test in tabpanel with a case when `items=[null]`. To fix:
```suggestion
    return this.option('items')[index]?.visible ?? true;
```

Reply:
[eb9495e](https://github.com/DevExpress/DevExtreme/pull/28128/commits/eb9495e5a1192db390e67834c0ed4c2764c2b9d4)